services/s3/src/main/java/software/amazon/awssdk/services/s3/S3Configuration.java

```diff
@@ -355,6 +376,32 @@ public interface Builder extends CopyableBuilder<Builder, S3Configuration> {
          * </p>
          */
         Builder profileName(String profileName);
+
+        /**
+         * <p> Configures whether cross-region bucket access is enabled for clients using the configuration.
+         * <p>The following behavior is <i>currently</i> used when this mode is enabled:
```

Comment:
Nit: why do we need emphasize `<i>currently</i>`?

Reply:
removed